builtins/amp-ad.js

```diff
@@ -283,6 +283,10 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
+      if (!this.placeholder_) {
+        this.element.style.height = 0;
```

Comment:
`display: none` please.

It should only do this when resizing is allowed. @dvoytenko can point you at the relevant code.